aep/general/0134/aep.md.j2

```diff
@@ -310,17 +310,19 @@ message Book {
 }
 ```
 
+- The `update_mask` field in the request does not affect the behavior of the
+  `etag` field, as it is not a field _being_ updated.
+
 {% tab oas %}
 
 **Note:** OAS example not yet written.
 
 {% endtabs %}
 
-The `etag` field **may** be either [required][] or [optional][]. If it is set,
-then the request **must** succeed if and only if the provided etag matches the
-server-computed value, and **must** fail with an `ABORTED` error otherwise. The
-`update_mask` field in the request does not affect the behavior of the `etag`
-field, as it is not a field _being_ updated.
+- The `etag` field **must** be [optional][].
```

Comment:
What's the rationale for saying that APIs cannot make etags required for any resource?

Reply:
There's a discussion on the overall pr here: https://github.com/aep-dev/aeps/pull/240#issuecomment-2458243569. 

There's some good back and forth there, but I'll close this issue since I don't think it actually achieves my goal of making full clients easier to author.